packages/ui/src/components/msg/governance/deposit_proposal/index.tsx

```diff
@@ -5,9 +5,17 @@ import { formatNumber, formatToken } from '@/utils/format_token';
 import { PROPOSAL_DETAILS } from '@/utils/go_to_page';
 import Typography from '@mui/material/Typography';
 import { Trans, useTranslation } from 'next-i18next';
-import Link from 'next/link';
-import { FC, useMemo } from 'react';
+import Link, { LinkProps } from 'next/link';
+import { FC, ReactNode } from 'react';
 
+const LinkText: FC<{ href: LinkProps['href']; children: ReactNode }> = (props) => {
```

Comment:
```suggestion
const LinkText: FC<PropsWithChildren<{ href: LinkProps['href'] >> = (props) => {
```

Reply:
which one do you prefer for style? `children: ReactNode` or `PropsWithChildren`?

Reply:
I leave it up to you to decide, `PropsWithChildren` seems to be the more popular choice https://blog.logrocket.com/using-react-children-prop-with-typescript/